src/DataStructures.jl

```diff
@@ -69,6 +69,7 @@ module DataStructures
     include("accumulator.jl")
     include("classified_collections.jl")
     include("disjoint_set.jl")
+    const find_root = find_root!  # remove when deprecating find_root. See deprecations.jl
```

Comment:
just put this in `disjoint_sets.jl`

Reply:
I think i would put this in `deprecations.jl`
Because that is the file I check everytime I am about to make a minor release.